popup.js

```diff
@@ -59,22 +76,20 @@ chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
       Array.from(cookieValue).forEach((element)=>{
         
         element.addEventListener("change", (e) => {
-          const cookie = cookies[Number(e.target?.attributes?.[0]?.value)]
-          const protocol = cookie.secure ? "https:" : "http:";
-          const cookieUrl = `${protocol}//${cookie.domain}${cookie.path}`;
-          console.log(e.target.value);
-
-
+          const {name, storeId, secure, domain, path, expirationDate} = cookies[Number(e.target?.attributes?.[0]?.value)]
+          const protocol = secure ? "https:" : "http:";
+          const cookieUrl = `${protocol}//${domain}${path}`;
           chrome.cookies.remove({
             url: cookieUrl,
-            name: cookie.name,
-            storeId: cookie.storeId
+            name: name,
+            storeId: storeId
           })
-          chrome.cookies.set({
-            url:cookieUrl,
-            name:cookie.name,
-            value:e.target.value,
-            storeId: cookie.storeId
+          updateCookie({
+            url: cookieUrl,
+            value: e.target.value,
+            name,
+            storeId,
+            expirationDate
```

Comment:
![image](https://user-images.githubusercontent.com/33455014/140766659-ab3afd2a-d61f-47f3-adff-295c633ee9e2.png)
Add ALL this logic to updateCookie function